### PR TITLE
chore: update tools

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ lint: lint-golang lint-jsonnet lint-shell
 
 .PHONY: lint-golang
 lint-golang: $(GOLANGCI_LINT)
-	$(GOLANGCI_LINT) run ./... --fix
+	$(GOLANGCI_LINT) run ./... --fix --timeout 2m
```
[the remark this answers is not in the document]
Comment (simonpasquier) [reply Makefile:31]: A [previous run](https://github.com/rhobs/observability-operator/actions/runs/10814939537/job/30002595801) of the lint job failed after 1m and reported:

````
/home/runner/work/observability-operator/observability-operator/tmp/bin/golangci-lint run ./... --fix
level=error msg="Running error: context loading failed: failed to load packages: failed to load packages: failed to load with go/packages: context deadline exceeded"
level=error msg="Timeout exceeded: try increasing it by passing --timeout option"
````

I'm not sure if this change helped or not.

[the remark this answers is not in the document]
Comment (simonpasquier) [reply Makefile]: Trying again without the timeout change.

[the remark this answers is not in the document]
Comment (simonpasquier) [reply Makefile]: Ok figured out that the local `.golangci.yml` used an old option for timeout which was set to 10m. I've adjusted the config.

**Review by marioferh — COMMENTED** [review]
/lgtm

**Pull request merged**